src/views/Compose.vue

```diff
@@ -878,6 +892,13 @@ export default {
     onRequestGroupSaved: function(requestGroupId) {
       this.$router.push({ name: 'requestgroupDetail', params: { id: requestGroupId } });
     },
+    showFractionalRate: function(target) {
+      if (target.type === 'ORBITAL_ELEMENTS' && !target.scheme.includes('MAJOR_PLANET')) {
+        return true;
+      }
+      delete target.extra_params['fractional_ephemeris_rate']
+      return false;
+    },
```

Comment:
This works fine but I think it would make more sense as a computed property. https://vuejs.org/v2/guide/computed.html#Computed-Caching-vs-Methods